common/components/WalletDecrypt/PrivateKey.jsx

```diff
@@ -26,7 +26,9 @@ export default class PrivateKeyDecrypt extends Component {
   render() {
     const { key, password } = this.props.value;
     const fixedPkey = fixPkey(key);
-    const isValid = isValidPrivKey(fixedPkey.length);
+    const isValid =
+      isValidPrivKey(fixedPkey.length) ||
+      isValidEncryptedPrivKey(fixedPkey.length);
```

Comment:
I see that `isValidPrivKey` was implemented like this before you, but it's kinda weird that we send the length, but the function seems to indicate it would check the whole private key. And maybe in the future we add some additional checks to it (Like to reject non-alphanumeric characters, or something.)

I think we should change these two to take in the key and do `.length` themselves.